### PR TITLE
skip wrapping synchronous callbacks in `CallbackTo`

### PR DESCRIPTION
**Pull request by oyvindberg**
After trying out the scalajs-react flavour I thought these were just in the way. WDYT @rleibman ?

### CODE REVIEW

File: tests/react-integration-test/check-japgolly/r/react-dropzone/src/main/scala/typingsJapgolly/reactDropzone/components/ReactDropzone.scala [hunk 1]
```diff
@@ -30,11 +29,13 @@ object ReactDropzone {
...
     def accept(value: String | js.Array[String]): this.type = set("accept", value.asInstanceOf[js.Any])
     @scala.inline
-    def children(value: /* state */ DropzoneState => CallbackTo[Element]): this.type = set("children", js.Any.fromFunction1((t0: /* state */ DropzoneState) => value(t0).runNow()))
+    def children(value: /* state */ DropzoneState => Element): this.type = set("children", js.Any.fromFunction1(value))
     @scala.inline
     def disabled(value: Boolean): this.type = set("disabled", value.asInstanceOf[js.Any])
     @scala.inline
-    def getFilesFromEvent(value: /* event */ DropEvent => CallbackTo[js.Any]): this.type = set("getFilesFromEvent", js.Any.fromFunction1((t0: /* event */ DropEvent) => value(t0).runNow()))
+    def getFilesFromEvent(
+      value: /* event */ DropEvent => /* import warning: transforms.QualifyReferences#resolveTypeRef many Couldn't qualify Promise<Array<File | DataTransferItem>> */ _
```
Comment (rleibman) [inline tests/react-integration-test/check-japgolly/r/react-dropzone/src/main/scala/typingsJapgolly/reactDropzone/components/ReactDropzone.scala:37]: Is this warning something to be concerned about?

Comment (oyvindberg) [reply tests/react-integration-test/check-japgolly/r/react-dropzone/src/main/scala/typingsJapgolly/reactDropzone/components/ReactDropzone.scala:37]: Yeah, it's fine actually. When we added all the react integration tests we didn't do all the busy work with ensuring that all referenced types were actually available.

**Comment by oyvindberg** [comment]
I pinged you in the PR since you were part of adding it, and have used the scalajs-react flavour more than me. 100% of the cases where there were a prop like `render: Route => CallbackTo[ReactElement]` I had to needlessly wrap into `CallbackTo`, so I'm thinking that might be the common case and we should just simplify.

any idea?

**Comment by oyvindberg** [comment]
demo diff https://github.com/ScalablyTyped/ScalaJsReactDemos/commit/53d0d6d85191f0fa1889dd583d6b0e231b713687

**Pull request merged**